examples/differential-expression.md

```diff
@@ -1,23 +1,26 @@
 Find genes differentially expressed by the kidney at significance level 0.05:
+
 ```python
 >>> from os import environ
 >>> from hubmap_api_py_client import Client
 >>> client = Client(environ['API_ENDPOINT'])
 
->>> organ_name = client.select_organs().get_list()[1]['grouping_name']
->>> organ_genes = client.select_genes(where='organ', has=[organ_name], genomic_modality='rna', p_value=0.05)
->>> organ_genes_details = organ_genes.get_list()[0:10]
->>> organ_genes_details[0].keys()
-dict_keys(['gene_symbol', 'go_terms'])
+>>> modality = 'atac'
+>>> gene_symbol = client.select_genes(where='modality', has=[modality]).get_list()[10]['gene_symbol']
```

Comment:
Any particular reason for `[10]`?

Reply:
No, that looks like a typo that I will fix.  Thanks for catching.

Reply:
Please fix typo.